data/remnant/remnant ships.txt

```diff
@@ -892,37 +892,43 @@ ship "Merganser"
 		category "Interceptor"
 		licenses
 			Remnant
-		"cost" 6830000
-		"shields" 3000
-		"hull" 5000
+		"cost" 4830000
+		"shields" 3500
+		"hull" 3000
 		"required crew" 1
 		"bunks" 3
 		"cargo space" 11
 		"fuel capacity" 600
 		"ramscoop" 1.75
-		"mass" 80
+		"mass" 130
 		"drag" 1.8
 		"heat dissipation" .75
 		"gaslining" 1
-		"outfit space" 219
-		"shield generation" 2.8
+		"outfit space" 181
+		"weapon capacity" 60
+		"engine capacity" 60
```

Comment:
```suggestion
		"engine capacity" 64
```
Still less than the Berserker, Hawk, or either of the Marauder interceptors (even the non-engines variants).
Unlike the weapon space, for which 60 is a very strong value for an interceptor (only beaten by the Marauder Bounder), 60 engine space is if anything slightly on the low side of average for an interceptor.
This also resolves the theming point about both being multiples of ten.